src/components/ContentSidebar/ActivityFeed/task/Task.js

```diff
@@ -22,7 +22,7 @@ const TASK_COMPLETED = 'completed';
 const TASK_INCOMPLETE = 'incomplete';
 
 type Props = {
-    task_assignment_collection: TaskAssignments,
+    task_assignment_collection: TaskAssignments | SelectorItems,
```

Comment:
I'm not sure we should be passing in assignments as selectorItems solely to display the name in the pending task assignment. @JustinHoldstock @DanDeMicco curious to know your thoughts or if you have better suggestions on what to do here?

Reply:
Is there a better way to do this?

Reply:
I'm open to suggestions but this way requires the least amount of overhead. We could try to massage the selector item into the form of a taskassignment but then we would have to make some other fields optional most likely.